config/scenario_config.csv

```diff
@@ -5,31 +5,31 @@ SSP2EU-NPi-calibrate;calibrate,AMT;calibrate;;14;;;rcp45;;;0;NPi;1;2100;;9;;;;;;
+SSP2EU-PkBudg500;1,AMT;;;;;;rcp20;9;globallyOptimal;500;diffCurvPhaseIn2Lin;100;2080;;9;;;;;;;;;;;NDC;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;on;on;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix4;2025;;SSP2EU-NPi;SSP2EU-NPi;;SSP2EU-PkBudg500: This climate policy scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The stylized climate policy scenario assumes a peak budget of 500 Gt?CO2 on total CO2 emissions from 2015 to 2100. This is a 1.5??C scenario, peak warming is allowed to be well belowwell below 1.5??C, at median climate sensitivity but returns to values below 1.5??C in at least 67?% of scenarios by the end of the century.
 SSP2EU-PkBudg650;1,AMT;;;;;;rcp20;9;globallyOptimal;650;diffCurvPhaseIn2Lin;100;2080;;9;;;;;;;;;;;NDC;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;on;on;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix4;2025;;SSP2EU-NPi;SSP2EU-NPi;;SSP2EU-PkBudg650: This climate policy scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The stylized climate policy scenario assumes a peak budget of 650 Gt?CO2 on total CO2 emissions from 2015 to 2100. This is a 1.5??C scenario, peak warming is allowed to be at or slightly above 1.5??C, at median climate sensitivity but returns to values below 1.5??C in at least 67?% of scenarios by the end of the century.
-SSP2EU-PkBudg1050;1,AMT;;;;;;rcp26;9;globallyOptimal;1050;diffCurvPhaseIn2Lin;60;2100;;9;;;;;;;;;;;NDC;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;on;on;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix3;2025;;SSP2EU-NPi;SSP2EU-NPi;;SSP2EU-PkBudg1050: This climate policy scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The stylized climate policy scenario assumes a peak budget of 1150 Gt?CO2 on total CO2 emissions from 2015 to 2100. This is a weel below 2??C scenario, peak warming is allowed to be at or slightly above 2??C, at median climate sensitivity but returns to values below 2??C in at least 67?% of scenarios by the end of the century.
+SSP2EU-PkBudg1050;1,AMT;;;;;;rcp26;9;globallyOptimal;1050;diffCurvPhaseIn2Lin;60;2100;;9;;;;;;;;;;;NDC;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;on;on;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix3;2025;;SSP2EU-NPi;SSP2EU-NPi;;SSP2EU-PkBudg1050: This climate policy scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The stylized climate policy scenario assumes a peak budget of 1150 Gt?CO2 on total CO2 emissions from 2015 to 2100. This is a weel below 2??C scenario, at median climate sensitivity but returns to values below 2??C in at least 67?% of scenarios by the end of the century.
```

Comment:
well below, not weel below. Delete "but returns to values below 2°C" and replace "by the end of the century" with "during the whole century".